pytorch_toolkit/image_classification/tests/train_tests_custom_classification.py

```diff
@@ -23,9 +23,10 @@
 )
 
 class CustomClassificationTestCase(
+
```

Comment:
Tests for new models?

Reply:
Is it not enough to test train/val only for small mobilenet to reduce the testing time? For all models, training pipelines remain the same.

Reply:
we need to make sure that it is runnable at least

Reply:
Ok, added the rest of the models